doc/man3/EVP_KDF.pod

```diff
@@ -248,14 +251,16 @@ supported by the KDF algorithm.
 
 =head1 SEE ALSO
 
-L<EVP_KDF-SCRYPT(7)>
-L<EVP_KDF-TLS1_PRF(7)>
-L<EVP_KDF-PBKDF2(7)>
 L<EVP_KDF-HKDF(7)>
+L<EVP_KDF-KB(7)>
+L<EVP_KDF-KRB5KDF(7)>
+L<EVP_KDF-PBKDF2(7)>
+L<EVP_KDF-SCRYPT(7)>
 L<EVP_KDF-SS(7)>
 L<EVP_KDF-SSHKDF(7)>
-L<EVP_KDF-X963(7)>
+L<EVP_KDF-TLS1_PRF(7)>
 L<EVP_KDF-X942(7)>
+L<EVP_KDF-X963(7)>
```

Comment:
So here's a question from a maintainability point of view...  should we have this duplicated list, or should we simply have `L<OSSL_PROVIDER-default(7)>`?

Reply:
Changed to L<OSSL_PROVIDER-default(7)/Key Derivation Function (KDF)>

Reply:
Works for me :-)